linux/system/init.sls

```diff
@@ -6,6 +6,9 @@ include:
 {%- if system.pkgs|length > 0 %}
 - linux.system.package
 {%- endif %}
+{%- if system.cpu.governor is defined %}
```

Comment:
rename govenor.sls to cpu.sls and do this conditional:
```if system.cpu is defined```

Then you can check whether we want to set govenor in cpu.sls using ```if system.cpu.govenor is defined``` etc.